lib/cocoapods/command/update.rb

```diff
@@ -42,6 +44,14 @@ def initialize(argv)
           end
         end
 
+        unless excluded_pods.empty?
+          @pods ||= config.lockfile.pod_names.dup
+          skipping_non_installed_pods = (excluded_pods - @pods).any?
+          raise Informative, 'Trying to skip pods that are not installed' if skipping_non_installed_pods
```

Comment:
this message could be a bit more descriptive and provide what actually went wrong. e.g. "Attempted to exclude pods `A,B,C` which they are not currently installed.`

Reply:
@dnkoutso also thought about it but at the time thought I'd have to test each skipped Pod for belonging to the installed set but turns out a simple subtract of two sets will do.
I did it slightly different from the way it's done for the [non installed Pods that are being updated ](https://github.com/CocoaPods/CocoaPods/blob/master/lib/cocoapods/command/update.rb#L57) because with the message I have it's easier to reuse the message and just pluralize two words.
If you value the consistency more though, I don't mind changing it to be like the other one.